path_config.go

```diff
@@ -167,24 +167,7 @@ func (b *azureSecretBackend) pathConfigWrite(ctx context.Context, req *logical.R
 		return nil, err
 	}
 
-	resp := addAADWarning(nil, config)
-
-	return resp, nil
-}
-
-const aadWarning = "This configuration is using the Azure Active Directory API which is being " +
-	"removed soon. Please migrate to using the Microsoft Graph API using the " +
-	"use_microsoft_graph_api configuration parameter."
-
-func addAADWarning(resp *logical.Response, config *azureConfig) *logical.Response {
-	if config.UseMsGraphAPI {
-		return resp
-	}
-	if resp == nil {
-		resp = &logical.Response{}
-	}
-	resp.AddWarning(aadWarning)
-	return resp
+	return nil, err
```

Comment:
We'll want to remove the ability to set `use_microsoft_graph_api` in [`pathConfigWrite`](https://github.com/hashicorp/vault-plugin-secrets-azure/blob/main/path_config.go#L147-L151). Right now if you set its value to false, the plugin will panic when trying to read credentials because the MS graph API client is nil.

I'd also say we can completely remove [the field](https://github.com/hashicorp/vault-plugin-secrets-azure/blob/main/path_config.go#L37) from `azureConfig`. This will also mean removing its usage from other locations in the code.

Reply:
Thank you @austingebauer. I just pushed another PR where I removed the useMsGraphApi field completely and its usages. When you get a chance, could you please review it again? Thank you!!

Reply:
Thanks and looks good! One last thing for this -- we should also remove the field from the path at [path_config.go#L74-L78](https://github.com/hashicorp/vault-plugin-secrets-azure/blob/removeaadgraphcode/path_config.go#L74-L78) since we're not parsing it anymore.